pkg/ipcache/metadata_test.go

```diff
@@ -1457,18 +1457,72 @@ func BenchmarkManyCIDREntries(b *testing.B) {
 	for _, v := range cidrs {
 		cidrLabels[v] = labels.GetCIDRLabels(v.AsPrefix())
 	}
+	half := len(cidrs) / 2
+	removeHalf := cidrs[:half]
+	insertHalf := cidrs[half:]
 
+	revsLen := 1024
+	revs := make([]uint64, 0, revsLen)
 	var i int
+
 	b.ResetTimer()
 	b.ReportAllocs()
 	for b.Loop() {
+		mu := make([]MU, 0, len(cidrLabels))
```

Comment:
this is a somewhat chunky allocation inside of the hot loop - could move it out?

Reply:
For the benchmark, we need to be able to reset contents of the slice, so unsure how to avoid this.

Reply:
I think you could pre-allocate scratch space big enough for that; but it's a bit of a hassle. If the benchmark works to detect the improved perf, it's probably good enough, even if the absolute numbers are skewed by the allocations.